clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareMembershipManager.java

```diff
@@ -117,6 +117,14 @@ public String rackId() {
         return rackId;
     }
 
+    @Override
+    public NetworkClientDelegate.PollResult poll(long currentTimeMs) {
+        if (state == MemberState.RECONCILING) {
+            maybeReconcile(true);
+        }
+        return NetworkClientDelegate.PollResult.EMPTY;
+    }
+
```

Comment:
ShareMembershipManager will be affected if we add a new argument to `maybeReconcile`, so I override the `poll` method for it to keep the original logic.